downloads.zh.md

```diff
@@ -18,120 +18,134 @@ $( document ).ready(function() {
 
 {% toc %}
 
-## 最新稳定版 (v{{ site.FLINK_VERSION_STABLE }})
-
-Apache Flink® {{ site.FLINK_VERSION_STABLE }} 是我们最新的稳定版本。
-
-使用 Apache Flink [不需要](faq.html#how-does-flink-relate-to-the-hadoop-stack)安装 Apache Hadoop。对于使用 Flink 而没有用到任何 Hadoop 组件的用户，我们建议使用不包含 Hadoop 库的包。
-
-如果你计划将 Apache Flink 与 Apache Hadoop 一起使用（在 YARN 上运行 Flink ，连接到 HDFS ，连接到 HBase ，或使用一些基于 Hadoop 文件系统的 connector ），请选择包含匹配的 Hadoop 版本的下载包，或使用不带 Hadoop 版本的包并[设置 HADOOP_CLASSPATH 环境变量](https://ci.apache.org/projects/flink/flink-docs-stable/ops/deployment/hadoop.html)。
-
-### 二进制包
-
-<table class="table table-striped">
-<thead>
-    <tr>
-    <th></th> <th>Scala 2.11</th> <th>Scala 2.12</th>
-    </tr>
-</thead>
-<tbody>
-    {% for binary_release in site.stable_releases %}
-    <tr>
-    <th>{{ binary_release.name }}</th>
-    {% if binary_release.scala_211 %}
-    <td><a href="{{ binary_release.scala_211.url }}" class="ga-track" id="{{ binary_release.scala_211.id }}">Download</a> (<a href="{{ binary_release.scala_211.asc_url }}">asc</a>, <a href="{{ binary_release.scala_211.sha512_url }}">sha512</a>)</td>
-    {% else %}
-    <td>Not supported.</td>
-    {% endif %}
-
-    {% if binary_release.scala_212 %}
-    <td><a href="{{ binary_release.scala_212.url }}" class="ga-track" id="{{ binary_release.scala_212.id }}">Download</a> (<a href="{{ binary_release.scala_212.asc_url }}">asc</a>, <a href="{{ binary_release.scala_212.sha512_url }}">sha512</a>)</td>
-    {% else %}
-    <td>Not supported.</td>
-    {% endif %}
-    </tr>
-    {% endfor %}
-</tbody>
-</table>
-
-### 源码包
-<p>使用以下任一软件包查看源代码或自行构建 Flink：</p>
-
-{% for source_release in site.source_releases %}
-<div class="list-group">
-  <!-- Source -->
-  <a href="{{ source_release.url }}" class="list-group-item ga-track" id="{{ source_release.id }}">
-    <!-- overrride margin/padding as the boxes otherwise overlap in subtle ways -->
-    <h4 style="margin-top: 0px; padding-top: 0px;"><span class="glyphicon glyphicon-download" aria-hidden="true"></span> <strong>{{ source_release.name }}</strong> Source Release</h4>
-  </a>
-   (<a href="{{ source_release.asc_url }}">asc</a>, <a href="{{ source_release.sha512_url }}">sha512</a>)
-</div>
-{% endfor %}
+Apache Flink® {{ site.FLINK_VERSION_STABLE }} is our latest stable release.
+
+An Apache Hadoop installation is [not required](faq.html#how-does-flink-relate-to-the-hadoop-stack) to use Apache Flink.
+For users that use Flink without any Hadoop components, we recommend the release without bundled Hadoop libraries.
```

Comment:
That's confusing. Remove since we don't have the option with bundled Hadoop any longer?

Reply:
Will change that 👍